packages/form-js-viewer/src/render/components/form-fields/Taglist.js

```diff
@@ -128,22 +128,22 @@ export default function Taglist(props) {
       label={ label }
       id={ prefixId(`${id}-search`, formId) } />
     <div class={ classNames('fjs-taglist', { 'disabled': disabled }) }>
-      {!disabled && loadState === LOAD_STATES.LOADED &&
+      { loadState === LOAD_STATES.LOADED &&
         <div class="fjs-taglist-tags">
           {
             values.map((v) => {
               return (
-                <div class="fjs-taglist-tag" onMouseDown={ (e) => e.preventDefault() }>
+                <div class={ classNames('fjs-taglist-tag', { 'disabled': disabled }) } onMouseDown={ (e) => e.preventDefault() }>
```

Comment:
```suggestion
                <div class={ classNames('fjs-taglist-tag', { disabled }) } onMouseDown={ (e) => e.preventDefault() }>
```

Reply:
This one I won't simplify as it would break if we ever renamed disabled. 

One disabled describes the css class while the other describes the condition for which it should be enabled. The fact that they are the same might change in the future, so for semantic and future-proofing reasons let's keep it this way.

Reply:
Well okay unlikely to change, but still I think it's good practice to decouple them.

Reply:
I think it's not really necessary to separate but ok :sweat_smile: 

but one other thing related this. Shouldn't prefix the class name with `fjs-`